docs/en/Authorization.md

```diff
@@ -236,6 +236,25 @@ When you write this code inside your permission definition provider, it finds th
 
 > Tip: It is better to check the value returned by the `GetPermissionOrNull` method since it may return null if the given permission was not defined.
 
+### Permission depending on a Condition
+
+Permission states can be managed according to a condition such as a Feature, Global Feature, or a custom condition. Any condition which implements `IPermissionStateProvider` can be added via calling `AddStateProvider()` method for **PermissionDefinition**.
+
+See an example below:
+
+```csharp
+myGroup.AddPermission("Author_Management")
+    // To depend on Global Feature
+    .AddStateProviders(new RequireGlobalFeaturesPermissionStateProvider("AuthorManagementFeature"));
+	// To depend on Feature
+    .AddStateProviders(new RequireFeaturesPermissionStateProvider("BookStore.AuthorManagement"));
+	// To depend on your custom implementation
+    .AddStateProviders(new MyCustomPermissionStateProvider());
+	
+```
```

Comment:
There is also global state providers ([see](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Authorization.Abstractions/Volo/Abp/Authorization/Permissions/AbpPermissionOptions.cs#L11)) which is not explained in this document.

Reply:
It's going to be long documentation, should we extract State Providers as a new document? @hikalkan ,

Or we can just place a part about how to implement & configure custom conditions end of the section

Reply:
Being long is not a problem. I will review your changes, thanks.